django_tables2/columns/templatecolumn.py

```diff
@@ -26,24 +27,27 @@ class TemplateColumn(Column):
     - *value*       -- value from `record` that corresponds to the current column
     - *default*     -- appropriate default value to use as fallback
     - *row_counter* -- The number of the row this cell is being rendered in.
+    - context variables pass as argument to column
 
     Example:
 
     .. code-block:: python
 
         class ExampleTable(tables.Table):
             foo = tables.TemplateColumn('{{ record.bar }}')
-            # contents of `myapp/bar_column.html` is `{{ value }}`
-            bar = tables.TemplateColumn(template_name='myapp/name2_column.html')
+            # contents of `myapp/bar_column.html` is `{{label}}: {{ value }}`
```

Comment:
can you add spaces around `label` to get consistent style with the other template variables in the example?